changelog.d/14592.bugfix

```diff
@@ -0,0 +1 @@
+Fix a long-standing bug where ?
```

Comment:
I suspect that this could result in missing device list changes (or maybe device updates), but I'm not sure how to word that in such a way that it will make sense. Suggestions would be appreciated!